packages/modular-scripts/src/__tests__/test.test.ts

```diff
@@ -195,135 +173,83 @@ describe('Modular test command', () => {
 
     beforeEach(() => {
       // Create random dir
-      randomOutputFolder = tmp.dirSync({ unsafeCleanup: true }).name;
+      randomOutputFolder = createModularTestContext();
```

Comment:
Do these tests need a mock modular repo or do they just need an empty temp directory? 
createModularTestContext writes files, copies files, creates symlinks - not huge performance impact but still potentially more than needed here?

Reply:
modular repo - with empty directory it complains that modular's tsconfig is not found